common/reviews/api/heft.api.md

```diff
@@ -65,6 +65,7 @@ export type CustomActionParameterType = string | boolean | number | ReadonlyArra
 
 // @beta
 export class HeftCommandLineUtilities {
```

Comment:
In principle ***every*** API is "utilized" to do something. 😋 The word `Utilities` adds 9 characters to an API name without conveying any meaningful information (beyond maybe that we called it `FooUtilities` because another API is already called `Foo` heheh).

I don't see a `HeftCommandLine` class -- maybe we can just drop the `Utilities` suffix?

Reply:
Renamed.